e2e-tests/tfm/cypress/support/utils/getInputByLabelText.js

```diff
@@ -0,0 +1,6 @@
+export default (label) => {
+  return cy
+    .contains('label', label)
+    .invoke('attr', 'for')
+    .then((id) => cy.get(`#${id}`));
```

Comment:
I have changed this to your suggestion but then have changed it back because there is a slight difference, `.contains('selector', 'text')` gets an element of that selector type with the text where as `.get('selector').contains('text')`, gets all types of that selector and then finds an element within one of then containing that text, so you are not guaranteed that the returned element actually is a label, but to be able to assume it has a 'for' attribute we really do need it to be a html 'label' and not say a span within the label that contains the text we are trying to fetch by